src/test/java/org/folio/circulation/domain/policy/PeriodTest.java

```diff
@@ -39,4 +47,30 @@ public void toMinutesWithUnknownInterval() {
     Period period = Period.from(10, "Unknown interval");
     assertEquals(0, period.toMinutes());
   }
-}
\ No newline at end of file
+
+  @Test
+  public void periodsWithDifferentIntervalNamesIsNotEqual() {
+    val oneMonth = months(1);
+    val thirtyOneDay = days(31);
+
+    assertNotEquals(oneMonth, thirtyOneDay);
+  }
+
+  @Test
+  public void twoPeriodsAreEqual() {
+    val oneMonth = months(1);
+    val oneMonthCopy = months(1);
+
+    assertEquals(oneMonth, oneMonthCopy);
+    assertThat(oneMonth.compareTo(oneMonthCopy), is(0));
+  }
+
+  @Test
+  public void canComparePeriodsWithDifferentIntervals() {
```

Comment:
I think a complimentary test for this would be where two periods that use different intervals are considered equal (assuming that is possible)

Reply:
Added a test to verify this.